crates/ruff_python_formatter/src/trivia.rs

```diff
@@ -1,7 +1,6 @@
-use ruff_text_size::{TextLen, TextRange, TextSize};
-use unic_ucd_ident::{is_xid_continue, is_xid_start};
+use ruff_text_size::TextSize;
 
-use ruff_python_whitespace::{is_python_whitespace, Cursor};
```

Comment:
Hmm. Why does git not recognize that this file was moved? It will make merging my PRs rather painful.

Reply:
Will resolve, but also happy to just wait for you to merge and handle the rebase on my end.

Reply:
The problem could be that I didn't move the whole file, I left behind some of the utility methods... But I can just move them all.